roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -107,20 +107,85 @@
   adfinis.maintenance.find:
     paths: "{{ ansible_facts.mounts | map(attribute='mount') | list }}"
     prune: "{{ linux_large_logfiles_prune + linux_additional_large_logfiles_prune }}"
-    xdev: yes
+    xdev: true
     type: file
     size: "+16M"
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.slurp:
+    src: "{{ logrotate_status_file }}"
+  changed_when: false
+  register: logrotate_output
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.set_fact:
+    logrotate_logs: "{{ logrotate_output.content | b64decode | split('\n') }}"
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.find:
+    paths: /var/log
+    recurse: true
+    patterns: "*.log"
+    file_type: file
+  register: all_logs
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.debug:
+    msg: "{{ all_logs.files | map(attribute='path') | list }}"
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.set_fact:
+    unmanaged_logs: "{{ unmanaged_logs | default([]) + [log_name] }}"
+  loop: "{{ all_logs.files }}"
+  loop_control:
+    loop_var: item
+
+- <<: *task
+  vars:
+    taskid: 10-037
+    name: "Logrotate: Are there large files that point to a too verbose service?"
+  ansible.builtin.set_fact:
+    logrotate_logs: "{{ (logrotate_status.content | b64decode).split('\n') }}"
+    log_name: "{{ item.path | basename }}"
```

Comment:
What's `item`? There is no loop here.

Reply:
I tested this with another playbook but ansible-lint blocked me for push... how to solve this?

- name: Find unlisted logs
  set_fact:
    unmanaged_logs: "{{ unmanaged_logs | default([]) + [log_name] }}"
  vars:
    logrotate_logs: "{{ (logrotate_status.content | b64decode).split('\n') }}"
    # Splitting logrotate status file content into a list of log filenames
  loop: "{{ all_logs.files }}"
  loop_control:
    loop_var: item
  vars:
    log_name: "{{ item.path | basename }}"
    log_file: "{{ log_name.split('.')[0] }}"
    # Extracting log file name without extension

Reply:
You have two `vars` section in there. Only one is permitted.

AFAICT the first section isn't needed at all. `logrotate_logs` is already set as a fact; no need to redefine it as a var.